google/resource_compute_shared_vpc.go

```diff
@@ -26,6 +27,7 @@ func resourceComputeSharedVpc() *schema.Resource {
 			"service_projects": &schema.Schema{
 				Type:     schema.TypeSet,
 				Optional: true,
+				Computed: true,
```

Comment:
`service_projects` becomes `Computed` as these IDs can also be managed by `google_compute_shared_vpc_service_project_association` resources.